orchagent/pfc_detect_mellanox.lua

```diff
@@ -132,8 +124,8 @@ for i = n, 1, -1 do
                                 local pfc_rx_packets_string = '"pfc_rx_packets","' .. tostring(pfc_rx_packets) .. '","pfc_rx_packets_last","' .. tostring(pfc_rx_packets_last) .. '",'
                                 local storm_condition_string = '"pfc_duration","' .. tostring(pfc_duration) .. '","pfc_duration_last","' .. tostring(pfc_duration_last) .. '",'
```

Comment:
Also instead of HDEL redis operation, could we change to set to reduce the time?

Reply:
Do you mean line 120-121?
I suggest keeping as it is
This happens very rarely. Only if we receive PFC storm covering 99% of polling interval.
In addition, this is old code.